Dockerfile

```diff
@@ -4,7 +4,7 @@
 
 FROM golang:1.15-alpine AS build
 
-# Optionally set HUGO_BUILD_TAGS to "extended" when building like so:
+# Optionally set HUGO_BUILD_TAGS to "extended" or "nodeploy" when building like so:
```

Comment:
`HUGO_BUILD_TAGS` is a comma-separated list of tags.  `extended` and `nodeploy` are not mutually exclusion.

Reply:
I can remove it from the Dockerfile if you'd like or change the copy

Reply:
Just leave it as it is now. We should not have this info spread out into ... hundreds of files, and I'm not even sure the Mage specific env var is the right form of documentation either.

But for now, leave this PR as it is. It is fine for now.

Reply:
Okay, sounds good :+1: